frontend/cypress/e2e/firearm-identification.cy.js

```diff
@@ -0,0 +1,63 @@
+describe('Firearm Identification', () => {
+  it('should identificate real firearm', () => {
+    cy.visit('/accueil')
+    cy.getByDataTestid('identification')
+      .contains('J’ai déjà mis mon arme en sécurité, je veux l’identifier')
+      .click()
+    cy.url()
+      .should('contain', '/instructions')
+    cy.url().should('contain', '/instructions')
+    cy.contains('h3', 'Pour un résultat optimal')
+    cy.contains('span', 'canon vers la droite')
+
+    cy.getByDataTestid('select-file').as('fileInput')
+    cy.intercept('POST', '/api/upload').as('upload')
+    cy.get('@fileInput').selectFile('./cypress/images/pistolet-semi-auto.jpg', { force: true })
+    cy.wait('@upload').then(({ response }) => {
+      expect(response.statusCode).to.eq(200)
+    })
+    cy.getByDataTestid('go-on-and-follow').click()
+    cy.url().should('contain', '/guide-identification/informations-complementaires')
+    cy.getByDataTestid('explanation').should('contain', 'questions supplémentaires')
+    cy.getByDataTestid('go-on-and-follow').click()
+    cy.getByDataTestid('go-on-and-follow').should('have.attr', 'disabled')
+    cy.contains('cartouches').first().click()
+    cy.getByDataTestid('go-on-and-follow').should('not.have.attr', 'disabled')
+    cy.getByDataTestid('go-on-and-follow').click()
+    cy.url().should('contain', '/guide-identification/resultat-final')
+    cy.getByDataTestid('arm-category').should('contain', 'Catégorie B')
+    cy.getByDataTestid('return-to-home-end').click()
+    cy.url().should('contain', '/accueil')
+  })
+
+  it('should identificate dummy firearm', () => {
+    cy.visit('/accueil')
+    cy.getByDataTestid('identification')
+      .contains('J’ai déjà mis mon arme en sécurité, je veux l’identifier')
+      .click()
+    cy.url()
+      .should('contain', '/instructions')
+    cy.url().should('contain', '/instructions')
+    cy.contains('h3', 'Pour un résultat optimal')
+    cy.contains('span', 'canon vers la droite')
+
+    cy.getByDataTestid('select-file').as('fileInput')
+    cy.intercept('POST', '/api/upload').as('upload')
+    cy.get('@fileInput').selectFile('./cypress/images/pistolet-semi-auto.jpg', { force: true })
+    cy.wait('@upload').then(({ response }) => {
+      expect(response.statusCode).to.eq(200)
+    })
+    cy.getByDataTestid('go-on-and-follow').click()
+    cy.url().should('contain', '/guide-identification/informations-complementaires')
+    cy.getByDataTestid('explanation').should('contain', 'questions supplémentaires')
+    cy.getByDataTestid('go-on-and-follow').click()
```

Comment:
ajouter
cy.url().should('contain', '/guide-identification/munition-type')

Reply:
Correction faite